### PR TITLE
fix: icon not clickable issue

### PR DESCRIPTION
**Pull request by jaspalsingh-aot**
# Description
Fix Side nave bar icon click issue

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Backend](https://nr-site-registry-137-backend.apps.silver.devops.gov.bc.ca)
- [Frontend](https://nr-site-registry-137-frontend.apps.silver.devops.gov.bc.ca)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-site-registry/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-site-registry/actions/workflows/merge.yml)

### CODE REVIEW

**Review by bolny — APPROVED** [review]
If there's a way to add a unit test for the click issue (I understand this isn't always possible with frontend work) I think we should have it, otherwise LGTM.

**Comment by jaspalsingh-aot** [comment]
> If there's a way to add a unit test for the click issue (I understand this isn't always possible with frontend work) I think we should have it, otherwise LGTM.

I’ve optimized the code and resolved a bug, but I understand that testing frontend interactions can be a bit more complex. I’d be happy to create a separate ticket to address the test cases, as they may require additional effort. Please let me know  that works.

**Review by bolny — DISMISSED** [review]

**Review by midhun-aot — CHANGES_REQUESTED** [review]
We need to discuss before merging this change.

**Review by midhun-aot — APPROVED** [review]

**Pull request merged**
